### DISCUSSION TITLE
Release 14.0.0 webinterface passt nicht zur Version

### DISCUSSION THREAD

**Discussion by sammykd** [jomjol/AI-on-the-edge-device#1975] [Q&A] [answered]
Hallo, bis jetzt läuft die aktuelle 14.0.0 sehr flüssig, danke für die super Arbeit. 
 Im Log taucht aber eine Unstimmigkeit zum Webinterface auf. Ich habe die Manuelle installation von der zip Datei her gewählt.
Folgendes steht im Log    
  `04T16:55:30 <WRN> [MAIN] Web UI version (?) does not match firmware version (fa69272+) !
[0d00h00m11s] 2023-02-04T16:55:30 <WRN> [MAIN] Please make sure to setup the SD-Card properly (check the documentation) or re-install using the AI-on-the-edge-device__update__*.zip!     `
Die Installation ist die Releaseversion 14.0.0 und nicht die 14.0.0 RC8. Es wurde alles neu aufgesetzt und auch der ESP vorab gelöscht.
Wie komme ich an die richtige WEB Version? Oder existiert hier noch ein Fehler?

**Comment by uwen70** [comment]
Ich habe im Webinterface das Update eingespielt und danach stimmten die beiden Versionen überein. Hatte aber vorher die RC8 laufen.

Alternativ in der SD Karte in /html/version.txt die Version eintragen. Bei mir steht drin:
```
Release: v14.0.0 (Commit: fa69272+)
fa69272
```

**Comment by sammykd** [comment]
Ok, danke für den Hinweis,.
Dann stimmt die Versionierung in der  manuellen Installationsdatei (ZIP) nicht. Ich probiere es morgen aus zu ändern.

**Comment by sammykd** [comment, marked as answer]
So bei beiden Systemen fehlt nach der Installotion über die Datei AI-on-the-edge-device__manual-setup__v14.0.0.zip die Datei version.txt im Path SD/HTML. Sie ist nicht in der AI-on-the-edge-device__manual-setup__v14.0.0.zip enthalten, wogegen sie sich in der AI-on-the-edge-device__remote-setup__v14.0.0.zip und in der AI-on-the-edge-device__update__v14.0.0.zip befindet, ich habe sie aus einer der beiden anderen Dateien entpackt und auf die Systeme gebracht, jetzt ist alles OK.

**Comment by caco3** [comment]
> So bei beiden Systemen fehlt nach der Installotion über die Datei AI-on-the-edge-device__manual-setup__v14.0.0.zip die Datei version.txt

Stimmt, da werden die falschen Dateien verwendet.
Ich werde das gleich im AI-on-the-edge-device__manual-setup__v14.0.0.zip korrigieren.

@jomjol, @slider0007 Zur Info.

> Alternativ in der SD Karte in /html/version.txt die Version eintragen.

Bitte macht das nicht so! Nicht nur die version.txt ist anders!

**Comment by sammykd** [comment]
Ok, was heisst dass?
 Das ich das ganze Verzeichnis HTML löschen und neu beschreiben muss? Oder funktioniert das ggf. Einfacher mit der update Datei? Oder ist die ganze Zip nicht OK. Bis jetzt tauchen keine Probleme auf.

**Reply by caco3** [reply]
mach einfach ein normales update, das ganze Web UI wird dann neu geschrieben.

**Reply by sammykd** [reply]
Ok, danke

**Reply by sammykd** [reply]
Sooo, habe jetzt auf Gas und der Wasseruhr ein Update auf V14.0.1 gemacht, bis jetzt läuft erst mal alles.

**Reply by andgosmart** [reply]
Hi zusammen, ich frage mal kurz nach da ich bei 14.0.3 über OTA auch einen Hinweis habe, das das Web interface nicht passt. Und tatsächlich ist das bei Version 13.0.8 geblieben. Ich habe nun mehrfach ein OTA Update gemacht, jedoch ändert sich das nicht. Soll ich nochmal frisch löschen und ein komplette Neuinstallation machen, oder eher die Cam an den Rechner nehmen und so ein Update machen, ohne OTA?

**Reply by sammykd** [reply]
Hallo ich habe bei der Version 14.0.0 das Problem gehabt, da in dem Paket einige falsche Dateien waren. Habe dann die V14.0.1 ZIP ......UPDATE.ZIP über OTA eingespielt und danach gab es die V14.0.3 auch wieder die ......UPDATE.ZIP über OTA eingespielt. Bei meinen beiden Systemen ist jetzt alles OK mit der Webseite. Ursprünglich hatte ich aber bei der Fehlerhaften V14.0.0 bootloader.bin, partition.bin und auch die firmware.bin per USB NEU geflascht, ob das Auswirkungen hatte, das es jetzt bei mir funktioniert kann ich nicht sagen. Würde also versuchen erst mal die V14.0.3 ....UPDATE.ZIP PER OTA zu übertragen, geht das dann immer noch nicht, kannst Du ja noch die firmware.bin über OTA übertragen und dann noch mal die ......updat.zip. Hilft das alles nix dann würde ich neu aufsetzen.

**Reply by caco3** [reply]
Ja, bei 14.0.0-14.0.2 waren noch einige Dateien falsch gepackt, deshalb kam es zum 14.0.3.

Bei Problemen oder Verdacht einfach noch mals neu ein OTA machen, es werden dann alle Dateien neu geschrieben.
Falls danach immer noch Dateien falsch sind, sehe ich folgende mögliche Gründe:
 - Browser cacht die Daten, obwohl er nicht sollte -> im private Moder oder anderem Browser mal testen
 - Die SD-Karte ist korrupt. Das sieht man meist nur im USB-Log.
